prow/plugins/label/label.go

```diff
@@ -42,13 +42,17 @@ func init() {
 
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
 	// The Config field is omitted because this plugin is not configurable.
-	return &pluginhelp.PluginHelp{
-			Description: "The label plugin provides commands that add or remove certain types of labels. Labels of the following types can be manipulated: 'area/*', 'committee/*', 'kind/*', 'priority/*' and 'sig/*'.",
-			WhoCanUse:   "Anyone can trigger this plugin on a PR.",
-			Usage:       "/[remove-](area|committee|kind|priority|sig) <target>",
-			Examples:    []string{"/kind bug", "/remove-area prow", "/sig testing"},
-		},
-		nil
+	pluginHelp := &pluginhelp.PluginHelp{
+		Description: "The label plugin provides commands that add or remove certain types of labels. Labels of the following types can be manipulated: 'area/*', 'committee/*', 'kind/*', 'priority/*' and 'sig/*'.",
+	}
+	pluginHelp.AddCommand(pluginhelp.Command{
+		Usage:       "/[remove-](area|committee|kind|priority|sig) <target>",
+		Description: "Applies certain type of label.",
```

Comment:
Applies or removes a label from one of the recognized types of labels.

Reply:
done